src/web/error.rs

```diff
@@ -84,10 +84,14 @@ impl ValidationError {
 #[derive(Debug, Fail)]
 pub enum ValidationErrorKind {
     #[fail(display = "{}", _0)]
-    FromDetails(String, RequestErrorLocation, Option<String>),
+    FromDetails(String, RequestErrorLocation, Option<String>, Option<Tags>),
```

Comment:
Do we still need to pass Tags through everywhere now that the middleware's handling things? Removing it all would be a nice cleanup

Reply:
If we want to add or alter tags in the future, yes.

Reply:
Thinking about this more: I agree that we can probably do a bit of cleanup here and remove some of the pass-thru tag stuff. We can also pull from the `request.extensions().get::<Tags>()` and pass that along if need be.